app/views/exam_templates/assign_errors.html.erb

```diff
@@ -207,21 +207,20 @@
         <div class="side_header">
           <h2><%= t('exam_templates.assign_errors.document_info') %></h2>
         </div>
-        <%= form_tag(nil, id: 'info_form', class:'side_header') do %>
+        <%= form_for :fix_error,
+                     url: fix_error_assignment_exam_templates_path(@assignment),
+                     html: { multipart: true, size: 1, class: 'side_header' } do |f| %>
```

Comment:
I'm not sure why you keep using `multipart` for forms. It really shouldn't be necessary.